### PR TITLE
Bug 1907793:  Show support info in VM template details

### PR DESCRIPTION
**Pull request by rawagner**
depends on https://github.com/openshift/console/pull/7522

### CODE REVIEW

**Comment by rawagner** [comment]
Follows what we have in dev perspective @matthewcarleton 
Screenshot from 2020-12-15 10-15-07


Screenshot from 2020-12-15 10-15-44

**Comment by matthewcarleton** [comment]
Should it say just "full"? Where did the "full" come from in general? I don't think we had that in any designs did we?

**Comment by rawagner** [comment]
We didnt, support level (Full/Community) is coming from annotation on the template - see https://issues.redhat.com/browse/CNV-7742

**Comment by matthewcarleton** [comment]
ah ok - it would always say "Red Hat - full" though right? Not just "full"?

**Comment by rawagner** [comment]
Changed to always show `<provider> - <support_level>` (before I had only `<support_level>` if there was just one)

File: frontend/packages/kubevirt-plugin/src/components/vm-templates/label.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.kv-template-support__label {
```
Comment (matthewcarleton) [inline frontend/packages/kubevirt-plugin/src/components/vm-templates/label.scss:1]: you could write this so there's only a margin if there's more than one

**Comment by yaacov** [comment]
/close

CI failing, will continue in #7644

**Pull request closed without merging**